dockstore-webservice/src/main/java/io/dockstore/webservice/resources/DockerRepoResource.java

```diff
@@ -866,6 +867,24 @@ public Tool getPublishedContainerByToolPath(
         }
     }
 
+    private void mutateBasedOnUserAgent(Entry entry, Consumer<Entry> mutator, ContainerRequestContext containerContext) {
```

Comment:
I think this is just changing the response to be backwards compatible with 1.6 CLI and below. That's pretty old so I think this can just be deleted.